docs/tutorial8.md

```diff
@@ -12,6 +12,22 @@ By leveraging SPIFFE and SPIRE, we demonstrated how to secure an application by
 
 We hope this tutorial provides you with a pattern example that can significantly harden the security posture of your deployed application.
 
+## Cleanup 
+
+To remove all the components installed during the workshop, execute the following commands:
+
+```shell
+envsubst < $TUTORIAL_ROOT/cloudnativesecuritycon-workload-identity-tutorial/resources/secure/apps/py.yaml | kubectl delete -f -
+envsubst < $TUTORIAL_ROOT/cloudnativesecuritycon-workload-identity-tutorial/resources/secure/vault/vault.yaml | kubectl delete -f -  
+envsubst < $TUTORIAL_ROOT/cloudnativesecuritycon-workload-identity-tutorial/resources/baseline/py.yaml | kubectl delete -f - 
```

Comment:
This does not seem to be needed

1. `db-config` is removed during the tutorial
2. The remainder of the resources are removed from the invocation of the secure py set of resources above